src/js/components/DateInput/doc.js

```diff
@@ -0,0 +1,60 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge } from '../../utils';
+
+export const doc = DateInput => {
+  const DocumentedDateInput = describe(DateInput)
+    .availableAt(getAvailableAtBadge('DateInput'))
+    .description('A control to input a single date or a date range.')
+    .usage(
+      `import { DateInput } from 'grommet';
+<DateInput id='item' name='item' />`,
+    )
+    .intrinsicElement('div');
+
+  DocumentedDateInput.propTypes = {
+    buttonProps: PropTypes.shape({}).description(
+      `Any properties to pass on to the underlying DropButton
+      when not inline and no format.`,
+    ),
+    calendarProps: PropTypes.shape({}).description(
+      'Any properties to pass on to the underlying Calendar.',
+    ),
+    defaultValue: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.arrayOf(PropTypes.string),
+    ]).description('The default date or date range value in ISO8601 format.'),
+    dropProps: PropTypes.shape({}).description(
```

Comment:
Should we specify that its default value is `{ align: { top: 'bottom', left: 'left' }` ?

Reply:
yes, added